source/moto_payments/index.html.md.erb

```diff
@@ -16,35 +16,67 @@ This means you can take payments:
 
-Email [govuk-pay-support@digital.cabinet-office.gov.uk](mailto:govuk-pay-support@digital.cabinet-office.gov.uk) and we'll:
+## Set up MOTO payments on a test account
 
-- review your [Payment Card Industry Data Security Standards (PCI DSS) assessment](/security/#payment-card-industry-pci-compliance) to check you can take payments over the phone and by post
-- enable MOTO payments in your service
+[Contact GOV.UK Pay support](mailto:govuk-pay-support@digital.cabinet-office.gov.uk) to ask us to enable MOTO payments on your test account.
 
-## Create a MOTO payment
+You do not need any provide us with any supporting information.
+
+## Set up MOTO payments on a live account
+
+How you set up MOTO payments on a live account differs depending on what payment service provider (PSP) you use.
+
+### Set up MOTO payments on a live account with Worldpay or ePDQ
+
+The current Government Banking PSP is Worldpay.
+
+1. Sign in to your [GOV.UK Pay account](https://selfservice.payments.service.gov.uk/) and go to __My services__.
+
+1. Select __Add a new service__ to add a new service for MOTO payments to your account.
+
+1. Contact your PSP and ask for a new merchant ID for MOTO payments.
+
+    You will use this ID when you [connect your new service to your PSP](/switching_to_live/#go-live).
+
+1. Send your [PCI DSS Self Assessment Questionnaire](https://www.pcisecuritystandards.org/document_library?category=saqs) and service name to [GOV.UK Pay support](mailto:govuk-pay-support@digital.cabinet-office.gov.uk).
+
+We will review your PCI DSS compliance. Your PCI DSS compliance requirements are different for MOTO payments compared to online payments.
 
-Make sure you've collected your user's:
+When we are sure that you have met your PCI DSS compliance requirements, we enable MOTO payments in your service. You can then take MOTO payments.
 
-- card number
-- card expiry date
-- card security code
-- name from their card
-- email address - if you’ve [enabled collecting your users’ email addresses](/integrate_with_govuk_pay/#collecting-your-users-39-billing-addresses)
+### Set up MOTO payments on a live account with Stripe
+
+Stripe is GOV.UK Pay’s current PSP.
+
+When using Stripe, you do not need to:
+
+- set up a new service for MOTO payments
+
+- ask Stripe for a merchant ID for MOTO payments because you can take MOTO and online payments using the same ID
+
+You only need to send your [PCI DSS Self Assessment Questionnaire](https://www.pcisecuritystandards.org/document_library?category=saqs) and service name to [GOV.UK Pay support](mailto:govuk-pay-support@digital.cabinet-office.gov.uk).
+
+We will review your PCI DSS compliance. Your PCI DSS compliance requirements are different for MOTO payments compared to online payments.
+
+When we are sure that you have met your PCI DSS compliance requirements, we enable MOTO payments in your service. You can then take MOTO payments.
+
+## Create a MOTO payment
```

Comment:
for the below section - not sure what 'go to the next_url' means. Might be one to have a developer look at?

Reply:
have added a link to a section that includes a definition of next url

Reply:
I'm just confused about what it's doing here - why do we tell people to 'go to the next_url' before they've even taken a payment? the next_url is where we redirect users after they've made and is in the API call that the service has made in step 2. I might just not understand the process here but it's not quite clear to me what a user is actually meant to do.

Reply:
@kbottla is this something you can advise on?

Reply:
@kbottla please just 🌴 😄 

@jonathanglassman @miriamalice I believe there's some confusion around `_url`s which is understandable! 

The `next_url` is what's returned by the API when a service asks for a new payment. By following that link they will be presented with the card payments page. 

Entirely separately, the `return_url` is passed in by the service when creating a new payment and that's what we redirect the user to after completing the entire journey. 

In business as usual Pay, the user is redirected to the `next_url` and that's how they're able to pay, in the case of Moto payments it would be the service that follows the `next_url` to the payment page to then fill in the details on behalf of the user.